docs/user-guide/creating-application/config-approval.md

```diff
@@ -0,0 +1,110 @@
+
+5. You can also view the approver(s) if you wish.
+
+    ![Figure 5: Viewing the Approver(s)](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/config-approval/approver-list-1.jpeg)
+
+{% hint style="info" %}
+The one who performs the edits cannot approve their own changes. A different user has to review and approve.
+{% endhint %}
+
+Only one draft can exist at time and you cannot create multiple drafts. In the top-right corner, you have the option to discard the draft if you don't wish to proceed with the edits you made. 
+
+---
+
+## Approving the Configuration
+
+Only a different super-admin user or someone (who is not amongst the editors of the draft), having `Configuration approver` access, can approve the changes made to the configuration files as shown below.
+
+![Figure 6: Authorized User Approving the Changes](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/config-approval/approval-screen.jpg)
+
+If required, you can check whether a user has approval rights by going to **Global Configurations** → **Authorization** (dropdown) → **User Permissions**.
+
+![Figure 7: Checking Permissions](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/config-approval/config-approver-1.jpg)
+
+
+---
+
+## Deploying with New Configuration Values
+
+Once the approver validates and approves your configuration changes, you can proceed to deploy your application with the updated configuration.
+
+1. Go to the **Build & Deploy** tab of your application.
+
+2. Click **Select Image** in the deployment flow. 
+
+    ![Figure 8: Build & Deploy Page](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/config-approval/deploy-pipeline.jpg)
+
+3. You can view an indicator at the bottom `Config Diff from Last Deployed`. Click **Review** to view the changes.
+
+    ![Figure 9: Checking Config Diff](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/config-approval/review-config-diff.jpg)
+
+{% hint style="info" %}
+If the new configuration is not yet approved, the changes made to the config would not be visible during deployment, it would show `No Config Diff from Last Deployed` at the bottom. In that case, check whether your changes are present in the live config or not. If your changes are absent, chances are your draft is either pending for approval or rejected (discarded).
+{% endhint %}
+
+4. Once you have verified the changes, you can click **Deploy**.
+
+    ![Figure 10: Deploying App using New Config](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/config-approval/deploy.jpg)
+
+{% hint style="info" %}
+If you don't wish to deploy with the new changes, you can choose `Last deployed config` from the available drop-down.
+{% endhint %}
+
+## Enabling/Disabling Config Protection
```

Comment:
Who can enable and disable protection? RBAC missing

Reply:
Please help me with that information, @abhibhaw. I will add it to the doc.
I am not sure I know the answer...